### PR TITLE
WT-6570 RTS to remove the history store updates with no stop timestamp

### PR DESCRIPTION
**Pull request by kommiharibabu**
The left over updates in the history store with no stop timestamp can lead
to ever-increasing history store size. Remove these extra updates in the
history store.

### CODE REVIEW

**Comment by agorrod** [comment]
@kommiharibabu I think I need a walk through of this change to really understand it. Could you put something in the calendar for next Monday?

File: src/txn/txn_rollback_to_stable.c [hunk 1]
```diff
@@ -421,7 +421,11 @@ __rollback_abort_row_ondisk_kv(
     __wt_row_leaf_value_cell(session, page, rip, NULL, vpack);
     prepared = vpack->tw.prepare;
     if (WT_IS_HS(S2BT(session))) {
-        if (vpack->tw.durable_stop_ts > rollback_timestamp) {
+        /*
+         * Abort the history store update with stop durable timestamp greater than stable timestamp
+         * or the updates with max stop timestamp.
```
Comment (agorrod) [inline src/txn/txn_rollback_to_stable.c:426]: Could you please expand this comment, adding "which implies that they are associated with prepared transactions" to the end?

Comment (kommiharibabu) [reply src/txn/txn_rollback_to_stable.c]: Thanks, fixed.

**Comment by kommiharibabu** [comment]
@agorrod I added the test that found the problem to the PR. I found the problem why the visibility is going wrong when the read timestamp is provided. Created WT-6583. Currently, I reduced the number of rows in the test to let it pass. I mentioned this test on that ticket also to let it increase once the issue is fixed.

**Pull request merged**